stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/AnnotatedClass.java

```diff
@@ -68,8 +63,10 @@
         FieldOption fieldOption = useStag != null ? useStag.value() : null;
         if (fieldOption == null) {
             useStag = findParentUseStagAnnotation(element);
-            // TODO: Do not default to FieldOption.SERIALIZED_NAME, instead set to null and fail after removing @GsonAdapterKey 1/30/17 [AR]
-            fieldOption = useStag != null ? useStag.value() : FieldOption.SERIALIZED_NAME;
+            fieldOption = useStag != null ? useStag.value() : null;
+
+            // The field option should never be null
+            Preconditions.checkNotNull(fieldOption);
```

Comment:
Hmmm, thinking about this now, I believe so. I don't think we have a model class in the sample that has a inherits from a non-annotated class, otherwise compilation would fail. I am going to remove this non null assertion and add in a test case for this scenario. Perhaps AnnotatedClass is named incorrectly 🤔